ublox/src/ubx_packets/cfg_val.rs

```diff
@@ -1288,3 +1362,49 @@ pub enum TModePosType {
     /// Lat/Lon/Height position
     LLH = 1,
 }
+
+#[derive(Debug, Default, Clone, Copy, PartialEq)]
+#[cfg_attr(feature = "serde", derive(serde::Serialize, serde::Deserialize))]
+pub enum NavSpgFixMode {
```

Comment:
Compatible with `CfgNav5FixMode` which should possibly be renamed and used twice as well

Reply:
renamed to `NavFixMode` and added entry in changelog under `Breaking`